opentutor_classifier/tests/fixtures/data/question3/training.csv

```diff
@@ -0,0 +1,11 @@
+exp_num,text,label
+0,37 by 40,good
+0,37 ft by 40 ft. reactangle,good
+0,The 37 x 40 rectangle,good
+0,37 feet by 40 feet,good
+0,37 x 40,good
+0,I think the 7 feet by 10 feet looks the most like a square,bad
+0,7 feet by 10 feet,bad
+0,fdasfasd,bad
+0,no,bad
+0,7 by 10,bad

```

Comment:
the hope here is that we should be able to expect BAD classification for bad answers that didn't need an extra line in the `bad-regex` list, e.g.

```
0,92 by 81,bad
```

Reply:
So we'll have a singular regular expression for good answers. This is always a trade-off between false negatives and false positives as well as what a teacher is willing to accept. This is a quirky expectation because there are four distinct possible rectangles and they've made the dimensions distinct (e.g., only one rectangle has a side of length 40ft). Thus, their regular ex for the good answer is: 37|40. I'll go with that for now and we can change as desired.

Reply:
Changed regex. These are the current examples in the test: "7 by 10" (bad), "38 by 39" (bad), "37x40" (good), "12x23" (bad), "45 x 67" (bad) which it passes.